lib/TaskRouterWorkerCapability.js

```diff
@@ -4,12 +4,14 @@ var TaskRouterCapability = require('./TaskRouterCapability');
 function TaskRouterWorkerCapability(accountSid, authToken, workspaceSid, workerSid) {
     TaskRouterCapability.call(this, accountSid, authToken, workspaceSid, workerSid);
 
-    this.reservationsUrl = this._baseUrl + "/Tasks/**";
+    this.tasksUrl = this._baseUrl + "/Tasks/**";
```

Comment:
Correct, but I'm not sure anyone would directly reference reservations_url since this variable's only purpose would be for generating access policies.

I can revert this if necessary though.